gui/pages/Content/Marketplace/MarketTools.js

```diff
@@ -13,7 +13,7 @@ export default function MarketTools(){
   const [loadingText, setLoadingText] = useState("Loading Toolkits");
```

Comment:
Change it to Loading Templates

Reply:
Adarsh said to change it to loading toolkits